### PR TITLE
TF-287 Fix bug replies do not preserve line breaks

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/156957444-a3d0e8cd-f878-4a7a-8fcc-ec71f97aa6a7.mov

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: model/lib/email/email_content.dart [hunk 1]
```diff
@@ -12,4 +12,19 @@ class EmailContent with EquatableMixin {
...
+      return content
+          .replaceAll("'", r"\'")
+          .replaceAll('"', r'\"')
+          .replaceAll('\r', '')
+          .replaceAll('\r\n', '')
+          .replaceAll('\n', '<br/>')
+          .replaceAll('\n\n', '<br/>');
+    }
+    return content;
+  }
```
Comment (chibenwa) [inline model/lib/email/email_content.dart:29]: I'd prefer to rely on a library to do text => HTML because now we are vulnerable to HTML injections

**Comment by chibenwa** [comment]
This is a real blocker to me BTW

Cc @hoangdat

File: model/lib/email/email_content.dart (hunk 1 above)
Comment (hoangdat) [reply model/lib/email/email_content.dart:29]: #309

Comment (hoangdat) [reply model/lib/email/email_content.dart:29]: Hi @chibenwa, back to that problem, we found that no library support us on that (build up the block quote html from plaintext). 
So how about we still keep our method like that, then sanitize the html before rendering?

Comment (hoangdat) [reply model/lib/email/email_content.dart:29]: please continue discuss on https://github.com/linagora/tmail-flutter/issues/309#issuecomment-1062698198

**Pull request merged**